core/internal/features_test.go

```diff
@@ -981,7 +981,7 @@ func TestIntegration_FluxMonitor_NewRound(t *testing.T) {
 	var job models.JobSpec
 	err := json.Unmarshal(buffer, &job)
 	require.NoError(t, err)
-	job.Initiators[0].InitiatorParams.Feeds = models.Feeds([]string{mockServer.URL})
+	job.Initiators[0].InitiatorParams.Feeds = cltest.JSONFromString(t, fmt.Sprintf(`["%s"]`, mockServer.URL))
```

Comment:
As a follow up, what does everyone think about creating a MustJSON(*testing.T, interface{})?

Reply:
Yeah, `JSONFromString` is an odd one out.